src/Servers/Kestrel/test/FunctionalTests/MaxRequestBufferSizeTests.cs

```diff
@@ -217,7 +218,7 @@ public async Task ServerShutsDownGracefullyWhenMaxRequestBufferSizeExceeded()
 
             var memoryPoolFactory = new DiagnosticMemoryPoolFactory(allowLateReturn: true);
 
-            using (var host = await StartWebHost(16 * 1024, data, false, startReadingRequestBody, clientFinishedSendingRequestBody, memoryPoolFactory.Create))
+            using (var host = await StartHost(16 * 1024, data, false, startReadingRequestBody, clientFinishedSendingRequestBody, memoryPoolFactory.Create))
```

Comment:
Lets add a try catch (OperationCanceledException) to the `host.StopAsync()` at line 275